sormas-e2e-tests/src/test/java/org/sormas/e2etests/steps/web/application/events/EditEventSteps.java

```diff
@@ -126,6 +126,12 @@ public EditEventSteps(WebDriverHelpers webDriverHelpers, EventService eventServi
           webDriverHelpers.clickOnWebElementBySelector(POPUP_SAVE);
           webDriverHelpers.waitUntilElementIsVisibleAndClickable(PERSON_DATA_SAVED);
         });
+
+    When(
+        "^I click on edit task icon$",
```

Comment:
we won't be able to reuse this one, and based on the comment left above, we might improve it in the following way:

Rename the step by adding also the name of the identified task. I know that it has no title, so another option would be to say "I click on edit icon of the last created task", and here to access the Task POJO, get a specific value and create the locator.

Reply:
changed name to this :) I click on edit task icon of the first created task